ezResx/Resource/ResourceMerger.cs

```diff
@@ -32,15 +32,8 @@ public List<ResourceItem> MergeResources(List<ResourceItem> solutionResources, L
                 return solutionResources;
             }
 
-            Console.ForegroundColor = ConsoleColor.Red;
-            Console.WriteLine("Translations will be lost:");
-            foreach (var lostResource in xlsxRes)
-            {
-                Console.WriteLine($"{lostResource.Key.File} {lostResource.Key.Name}");
-            }
-                
-            Console.ResetColor();
-            throw new DataLossException("Potential data loss");
+            throw new DataLossException("Translations will be lost:", xlsxRes);
```

Comment:
Ending a message with colon is  kind of silly :). "Translations will be lost." should suffice